reorder_python_imports.py

```diff
@@ -690,15 +690,21 @@ def main(argv=None):
             separate_from_import=args.separate_from_import,
             application_directories=args.application_directories.split(':'),
         )
-        if contents != new_contents:
-            retv = 1
+        retv = contents != new_contents
```

Comment:
this accidentally regressed this so only the last file would control the exit code -- I'm fixing this nbd